app.config.js

```diff
@@ -0,0 +1,12 @@
+import { sentryApi } from './src/config/secrets';
+
+export default {
+  hooks: {
+    postPublish: [
+      sentryApi?.config && {
+        file: 'sentry-expo/upload-sourcemaps',
+        config: sentryApi.config
+      }
+    ]
+  }
+};
```

Comment:
i wasn't quite sure, if this extends or overwrites. in my understanding it extends simmply this way.
otherwise we would need to do:
```
export default ({ config }) => ({
  ...config,
  hooks: {
    ...
  }
});
```

Reply:
great, thanks